user_docs/en/userGuide.t2t

```diff
@@ -2231,6 +2231,16 @@ Please see the [EcoBraille documentation ftp://ftp.once.es/pub/utt/bibliotecnia/
 | Toggle braille tethered to | A |
 %kc:endInclude
 
+++ Super Braille ++
+The SuperBraille device, mostly available in Taiwan, can be connected to by either USB or serial.

```

Comment:
Are there any links we need to/can provide here for USB drivers or the like? I get this thing isn't overly well documented (at least in English), so we can always add that later if you have no idea and someone asks for it.